### PR TITLE
Ventilator visualization upgrades

### PR DESCRIPTION
**Pull request by adelarue**
This PR goes through the ventilator page and aims to reduce the amount of text without reducing the total content, through formatting and other tricks.

### CODE REVIEW

**Comment by leobix** [comment]
Looks great! Makes it clearer and easier to use as a tool!

File: ventilators/shortage.py [hunk 1]
```diff
@@ -7,44 +7,13 @@
 
 # Defines the top of the page, where the current situation is displayed (no optimization yet)
 shortage = \
+    [] + \
```
Comment (galitLukin) [inline ventilators/shortage.py:10]: Why did you add this? If it is for padding, can you add paddingTop in style?

File: ventilators/transfers_visuals.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 import dash_html_components as html
 import dash_bootstrap_components as dbc
 
-from ventilators.utils import df_mod1_transfers, oneWeekFromNow, state_cols
+from ventilators.utils import df_mod1_transfers, oneWeekFromNow, state_cols, today
```
Comment (galitLukin) [inline ventilators/transfers_visuals.py:5]: you're not using this. did you mean to change the default dates to today? If you do make this change, please make it in both date pickers and delete the import and probably calculation of oneWeekFromNow

File: ventilators/transfers_visuals.py [hunk 3]
```diff
@@ -69,10 +128,10 @@
             [
                 dbc.Col(
                 [
-                    html.H6("By intelligent optimization via the ventilator allocation \
-                    model, we are able to reduce the shortage of ventilators in the states \
-                    substantially.",id="graph-vent-text"),
-                ]
+                    dbc.Alert("By optimizing ventilator allocation across states, we can quickly eliminate the shortage of ventilators in every state.",
```
Comment (galitLukin) [inline ventilators/transfers_visuals.py:131]: There's text on top of the table "Below you can view the inter-state...." Do you think it's a good idea to also put this text in an alert or in a card?

File: ventilators/shortage.py (hunk 1 above)
Comment (adelarue) [reply ventilators/shortage.py:10]: I added this to try something, and forgot about it

File: ventilators/transfers_visuals.py (hunk 2 above)
Comment (adelarue) [reply ventilators/transfers_visuals.py:5]: Removed today. I am conflicted about the default date value, need to discuss with @hwiberg, @leobix and the rest of the optimization team.

File: ventilators/transfers_visuals.py (hunk 3 above)
Comment (adelarue) [reply ventilators/transfers_visuals.py:131]: I made some changes along the lines of your suggestion. Let me know what you think :)
